### PR TITLE
Update access-props-using-this.props.spanish.md

### PR DESCRIPTION
**Pull request by dulcehc**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

File: curriculum/challenges/spanish/03-front-end-libraries/react/access-props-using-this.props.spanish.md [hunk 1]
```diff
@@ -4,14 +4,14 @@ title: Access Props Using this.props
 challengeType: 6
 isRequired: false
 videoUrl: ''
-localeTitle: Acceder a los apoyos usando this.props
+localeTitle: Acceder a los props usando this.props
 ---
 
 ## Description
-<section id="description"> Los últimos varios desafíos cubrieron las formas básicas de pasar accesorios a los componentes secundarios. Pero, ¿qué sucede si el componente secundario al que le está pasando una propuesta es un componente de clase ES6, en lugar de un componente funcional sin estado? El componente de clase ES6 utiliza una convención ligeramente diferente para acceder a accesorios. Cada vez que refiera a un componente de clase dentro de sí mismo, use <code>this</code> palabra clave. Para acceder a los apoyos dentro de un componente de clase, que Prefacio el código que se utiliza para acceder a ella con <code>this</code> . Por ejemplo, si un componente de clase ES6 tiene una propiedad llamada <code>data</code> , escribe <code>{this.props.data}</code> en JSX. </section>
```
[the remark this answers is not in the document]
Comment (dulcehc) [reply curriculum/challenges/spanish/03-front-end-libraries/react/access-props-using-this.props.spanish.md]: That's right, it's not a correct translation. Here props is translated as "apoyos" and it doesn't make sense. One solution could be not translate the word "props", the second one is translate it as "propiedades". Also, the complete sentence is not grammatically correct.

The text would be like:
Para acceder a <code>props</code> dentro de un class component, se precede el código que se utiliza para acceder a ella con <code>this</code>.

[the remark this answers is not in the document]
Comment (scissorsneedfoodtoo) [reply curriculum/challenges/spanish/03-front-end-libraries/react/access-props-using-this.props.spanish.md]: I'm not a Spanish speaker, so I'll let you decide which is best. Just happened to see the word "apoyos" again and thought I should ask.

Please go ahead and make whatever changes you feel are best when you have some time!

[the remark this answers is not in the document]
Comment (dulcehc) [reply curriculum/challenges/spanish/03-front-end-libraries/react/access-props-using-this.props.spanish.md]: I've also added the translations for the messages within the Challenge code.

**Comment by scissorsneedfoodtoo** [comment]
@dulcehc, everything LGTM! Thank you for helping us improve this section of the curriculum, and for making those additional changes so quickly. Looking forward to your next contribution! :+1: :+1:

**Pull request merged**